app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -21,6 +21,7 @@ const Notes = () => {
   const [searchTerm, setSearchTerm] = useState("");
   const [selectedNoteIds, setSelectedNoteIds] = useState([]);
   const [notes, setNotes] = useState(NOTES_TABLE_ROW_DATA);
+
   const { t } = useTranslation();
 
   const fetchNotes = async () => {
```

Comment:
@yedhink This function will no longer be required, as I have managed all the creation and deletion of notes in the state instead of API for this challenge. And if we remove this function, then setLoading function will not be required. 

But I saw a requirement mentioning `11.The application should have a loading state indicator.` in the instructions. Can you elaborate a little for what this loading state should be used for?

Reply:
Also, one more doubt, I saw the line `The dashboard in the design should show a list of dummy notes in a tabular format using the existing table styles from neetoUI.` in the challenge instructions and assumed that only requirements need to be implemented in this challenge with the exiting styles of the wheel app. 

Do correct me if I misunderstood the challenge and if the design of the web pages should completely replicate the Figma designs.

Reply:
@yedhink Any comments on this?

Reply:
>The function fetchNotes will no longer be required, as I have managed all the creation and deletion of notes in the state instead of API for this challenge. And if we remove this function, then setLoading function will not be required.

That's fine.

>But I saw a requirement mentioning 11.The application should have a loading state indicator. in the instructions. Can you elaborate a little for what this loading state should be used for?

That's only needed whenever there an action that might take time to complete. In such cases loaders can be added. If there are not cases in your app where such loading is required, then no loaders have to be added.

>in the challenge instructions and assumed that only requirements need to be implemented in this challenge with the exiting styles of the wheel app.

Not in styles of wheel app. The styles should strictly adhere to the default styles from https://neeto-ui.neeto.com/?path=/story/welcome--page.